docs/SETUP.md

```diff
@@ -1,75 +1,106 @@
+* Kernel version 4.0 or higher
+* Docker engine:
+    * Docker Enterprise Engine 17.06.2 or higher
+    * Docker Community Engine 17.06.2 or higher
+* [OverlayFS](https://docs.docker.com/storage/storagedriver/overlayfs-driver/) `overlay2` Docker daemon storage driver
+    1. Edit `/etc/docker/daemon.json`. If it does not yet exist, create it.
+    2. Assuming the file was empty, add the following contents:
+        ```
+        { "storage-driver": "overlay2" }
+        ```
+        **Note:** If you already have an existing JSON file, add only `"storage-driver": "overlay2"` as a key-value pair. Docker does not start if the `daemon.json` file contains badly-formed JSON.
 
-For more details, please see the official [supported architectures and platforms for containerized Splunk environments](https://docs.splunk.com/Documentation/Splunk/latest/Installation/Systemrequirements#Containerized_computing_platforms) as well as [hardware and capacity recommendations](https://docs.splunk.com/Documentation/Splunk/latest/Installation/Systemrequirements). 
+For more details, see the official [supported architectures and platforms for containerized Splunk software environments](https://docs.splunk.com/Documentation/Splunk/latest/Installation/Systemrequirements#Containerized_computing_platforms) as well as [hardware and capacity recommendations](https://docs.splunk.com/Documentation/Splunk/latest/Installation/Systemrequirements).
+Basic instructions to [deploy and run Splunk Enterprise inside containers](https://docs.splunk.com/Documentation/Splunk/latest/Installation/DeployandrunSplunkEnterpriseinsideDockercontainers) are also available.
 
-## Configure
-Before we can run the containers, we should pull it down from DockerHub. Run the following commands to pull the images into your local environment:
+## Installation
+Run the following commands to pull the latest images down from Docker Hub and into your local environment:
 ```
 $ docker pull splunk/splunk:latest
 $ docker pull splunk/universalforwarder:latest
 ```
 
 ## Run
-Before we stand up any containers, let's first create a network to enable communication between each of the services. This step is not necessary if you only wish to create a single, isolated instance.
+
+This section explains how to start basic standalone and distributed deployments. See the [Examples](EXAMPLES.md) page for instructions on creating additional types of deployments.
+
+### Standalone deployment
+
+Start a single containerized instance of Splunk Enterprise with the command below, replacing `<password>` with a password string that conforms to the [Splunk Enterprise password requirements](https://docs.splunk.com/Documentation/Splunk/latest/Security/Configurepasswordsinspecfile).
+
+```
+$ docker run -it -p 8000:8000 -e "SPLUNK_PASSWORD=<password>" -e "SPLUNK_START_ARGS=--accept-license" splunk/splunk:latest
+```
+
+**You successfully created a standalone deployment with `docker-splunk`!**
```

Comment:
Thinking out loud here, but should we bother explaining the env vars here? On one hand, I appreciate and like the quick-and-easy happy path CLI to get users started, but on the other hand the "Distributed deployment" section has more details about the env vars and why something like `SPLUNK_START_ARGS=--accept-license` is needed.

Reply:
Fair point. I can add the command breakdown to this section as well.